gui/src/components/logs/LogRow.svelte

```diff
@@ -1,26 +1,20 @@
+<script lang="ts" context="module">
+  export type GetComponentNameFunc = (id: string) => string | null;
+</script>
+
 <script lang="ts">
   import FormattedLogMessage from './FormattedLogMessage.svelte';
   import { shortTime } from '../../lib/time';
   import type { LogEvent } from '../../lib/logs/types';
   import { logStyleFromHash } from '../../lib/color';
 
-  export let processIdToName: Record<string, string>;
+  export let getComponentName: GetComponentNameFunc = (id) => null;
   export let event: LogEvent;
 
-  // SEE NOTE [LOG_COMPONENTS].
-  const streamToFriendlyName = (log: string): string => {
-    const [procId, stream] = log.split(':');
-    const procName = processIdToName[procId];
-    return procName ? (stream ? `${procName}:${stream}` : procName) : log;
-  };
-
-  const friendlyName = streamToFriendlyName(event.stream);
-
-  // SEE NOTE [LOG_COMPONENTS].
-  const trimLabel = (name: string) => name.replace(/(:err$)|(:out$)/g, '');
+  const componentName = getComponentName(event.stream) ?? event.stream;
 </script>
 
-<tr style={logStyleFromHash(friendlyName)}>
+<tr style={logStyleFromHash(componentName)}>
```

Comment:
Note: this should be updated in `ProcessList` to align the two color format inputs if they aren't already @brandonbloom , so that the same process has the same color in both panels.

Reply:
Nothing actually changed about this here, as best I can tell, so it should remain aligned with the ProcessList. However, I think _both_ of them should be updated to use the component id instead of the component name as the hash input value, so that the colors remain the same if a component is renamed (which is a power we will have soon, when I overhaul the manifest system - which i am working on now).

@jwmza Could you make that change?

Reply:
(the stream name is always equal to the component id now)

Reply:
Yeah, will do!